Web/AiiaClient/AiiaClientSecrets.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace Aiia.Sample.Services;
+
+public struct AiiaClientSecrets
```

Comment:
can this (AiiaClientSecrets) and AiiaAccessTokens get renamed to singular names?

Reply:
bump